tests/charts/test_scheduler.py

```diff
@@ -304,6 +304,38 @@ def test_logs_persistence_changes_volume(self, log_persistence_values, expected_
 
         assert {"name": "logs", **expected_volume} in jmespath.search("spec.template.spec.volumes", docs[0])
 
+    def test_scheduler_securityContexts_are_configurable(self):
+        docs = render_chart(
+            values={
+                "scheduler": {
+                    "securityContexts": {
+                        "pod": {
+                            "fsGroup": 1000,
+                            'runAsGroup': 1000,
```

Comment:
```suggestion
                            'runAsGroup': 2000,
```

Might be nice to use different ints here.

Reply:
tried to add some variation